docs/expressions.md

```diff
@@ -510,7 +510,20 @@ Group functions modify the OpenTSDB groups.
 ## addtags(set variantSet, group string) (seriesSet|numberSet)
 {: .exprFunc}
 
-Accepts a series and a set of tags to add to the set in `Key1=NewK1,Key2=NewK2` format. This is useful when you want to add series to set with merge and have tag collisions.
+Accepts a set and a set of tags to add to the set in `Key1=NewK1,Key2=NewK2` format. This is useful when you want to add series to set with merge and have tag collisions.
+
+## metatag(set variantSet, tagsCSV string, keyCSV string) (seriesSet|numberSet)
```

Comment:
I see now, it does not operate quite like I expected. Perhaps more examples with different usage patterns.